src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -110,6 +110,46 @@ def bitwise_xor(
     )
 
 
+@nameable_op
+def multinomial(
+    probs: NodeInput,
+    num_samples: NodeInput,
+    convert_type: str,
+    with_replacement: bool,
+    log_probs: bool,
+    global_seed: int = 0,
+    op_seed: int = 0,
+) -> Node:
+    """Return a node which generates a sequence of class indices sampled from the multinomial distribution.
+
+    :param probs: Tensor with probabilities of shape [class_size] or [batch_size, class_size].
+    :param num_samples: Tensor (scalar or 1D) with the lower bound on the range of random values to generate.
```

Comment:
```suggestion
    :param probs: Tensor with probabilities of shape [class_size] or [batch_size, class_size] and floating-point datatype.
    :param num_samples: Tensor (scalar or 1D) and integer datatype specifying the number of samples to draw from multinomial distribution.
```
I'm not sure whether description for num_samples is correct, + add info about datatype

Reply:
Good point, the description has been updated.